### PR TITLE
Add feature flag for inbound sso, update keep-alive call

### PR DESCRIPTION
**Pull request by hinzed1127**
## Description

This wires in a feature for inbound SSO (i.e., automatic login or logout of a `vets-api` session depending on the existence of an SSOe session), so we can toggle that independently of general SSOe session establishment via normal login interactions.

### CODE REVIEW

**Review by ncksllvn — APPROVED** [review]

File: src/platform/utilities/feature-toggles/featureFlagNames.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ export default Object.freeze({
   vaOnlineSchedulingPast: 'vaOnlineSchedulingPast',
   vaGlobalDowntimeNotification: 'vaGlobalDowntimeNotification',
   ssoe: 'ssoe',
+  ssoeInbound: 'ssoeInbound',
```
Comment (ericbuckley) [inline src/platform/utilities/feature-toggles/featureFlagNames.js:17]: @hinzed1127 I think this needs to be `ssoe_inbund` to match https://github.com/department-of-veterans-affairs/vets-api/pull/4202

Comment (hinzed1127) [reply src/platform/utilities/feature-toggles/featureFlagNames.js:17]: it looks like the flipper controller [converts it](https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/controllers/v0/feature_toggles_controller.rb#L14).

The client [docs](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/platform/tools/feature-toggles/#writing-good-feature-toggles) say to do camelCase, probably just going by language conventions in each

Comment (ericbuckley) [reply src/platform/utilities/feature-toggles/featureFlagNames.js:17]: oh cool, I had no idea

**Review by ericbuckley — APPROVED** [review]
👍

File: src/platform/utilities/feature-toggles/featureFlagNames.js (hunk 1 above)
Comment (patrickvinograd) [reply src/platform/utilities/feature-toggles/featureFlagNames.js:17]: The API uses a gem called olive_branch, which lets caller request conversion of JSON keys to camelCase based on the value of the "X-Key-Inflection" (or something like that) header. Was done to allow Node frontend and Rails backend to have as-expected inflection in their respective code domains.

**Comment by patrickvinograd** [comment]
Is this expected to both wire in the feature flag and solve the issue with keepalive?

**Comment by ericbuckley** [comment]
> Is this expected to both wire in the feature flag and solve the issue with keepalive, or are the changes to the keepalive speculative and we'll use the feature flag to test them in dev/staging?

Yeah, we're hoping that the addition of Content-Type header to the call will fix it, but we're not 100% because we can't test this locally.

**Review by patrickvinograd — APPROVED** [review]
LGTM. Corresponding feature flag definition in vets-api is merged.

**Review by billfienberg — APPROVED** [review]
Looks reasonable to me

**Pull request merged**